webapp/src/app.tsx

```diff
@@ -3971,22 +3975,31 @@ export class ProjectView
             return Promise.resolve(); // TODO cleanup
         }
         else {
-            return this.exitTutorialAsync()
-                .then(() => {
-                    let curr = pkg.mainEditorPkg().header;
-                    return this.loadHeaderAsync(curr);
-                }).finally(() => {
-                    core.hideLoading("leavingtutorial")
-                    this.postTutorialProgress();
-                })
-                .then(() => {
-                    if (pxt.appTarget.cloud &&
-                        pxt.appTarget.cloud.sharing &&
-                        pxt.appTarget.appTheme.shareFinishedTutorials) {
-                        pxt.tickEvent("tutorial.share", undefined, { interactiveConsent: false });
-                        this.showShareDialog(lf("Well done! Would you like to share your project?"));
-                    }
-                })
+            if (pxt.BrowserUtils.isSkillmapEditor()) {
+                return this.exitTutorialAsync()
+                    .finally(() => {
+                        core.hideLoading("leavingtutorial")
+                        this.postTutorialProgress();
```

Comment:
do we need this? we're posting the tutorial completed event above. could we just use the same IE check above?

Reply:
Can't use the same check because the IE one opens the home screen. I'll fix the progress event